docs/monitoring/infrastructure.md

```diff
@@ -37,15 +38,38 @@ When doing a clean installation, regardless of whether you're installing in cont
 
 ### The `setup-infra` command
 
-Setting up or changing which Infra App should receive infrastructure data and metrics from a particular machine can be done with the <b>setup-infra</b> command. To see which Infra Apps exist in your account visit Sematext Cloud -> Integrations -> Apps. You can choose any of the existing Infra Apps or create a new one. Once you do that you'll have the token of that particular App. To set up the token run the following command on each machine that you wish to update:
+Setting up or changing which Infra App should receive infrastructure data and metrics from a particular machine can be done with the <b>setup-infra</b> command.
+
+To see which Infra Apps exist in your account visit Sematext Cloud -> Integrations -> Apps. You can choose any of the existing Infra Apps or create a new one. 
+
+Once you do that you'll have the token of that particular App. To set up the token run the following command on each machine that you wish to update:
 
 <pre>sudo bash /opt/spm/bin/setup-infra --infra-token YOUR_INFRA_APP_TOKEN_HERE</pre>
 
+
+## Create an Infra App
+
+Creating a new Infra App is the same process as with any App in Sematext. In the Integrations view you choose Infra App, give it a name and click continue.
+
+<video style="display:block; width:100%; height:auto;" controls autoplay loop>
+  <source src="https://sematext.com/wp-content/uploads/2019/07/create-infra-app-1.mp4" type="video/mp4" />
+</video>
+
+You can then choose which operating system your machine are running. There are two simple setup commands you need to run. After that you're ready to go.
```

Comment:
Maybe instead "You can then choose..." be more precise about what is user actually doing here (otherwise it may sound like user is choosing some settings)? For example: "Next screen shows installation instructions which you'll have to follow to start using Infra App. Select the operating system running on your machines and run simple setup commands..."

Reply:
Edited.